pybikes/data/nextbike.json

```diff
@@ -1272,6 +1272,18 @@
                 "country": "DE"
             },
             "city_uid": 301
+        },
+        {
+            "domain": "lr",
```

Comment:
Domain should be `lp` for getting this network:

https://nextbike.net/maps/nextbike-live.xml?domains=lp